azext_iot/monitor/parsers/central_parser.py

```diff
@@ -45,7 +45,7 @@ def _add_central_issue(self, severity: Severity, details: str):
         )
 
     def parse_message(self) -> dict:
-        parsed_message = super(CentralParser, self).parse_message()
+        parsed_message = super(CentralParser, self).parse_message(include_component=False)
```

Comment:
plumbing this through the common args can avoid this, especially if component becomes useful for us to use later as we bring Components into central